### DISCUSSION TITLE
Prevents upgrades 24H2

### DISCUSSION THREAD

**Discussion by 4-FLOSS-Free-Libre-Open-Source-Software** [valinet/ExplorerPatcher#3807] [General]
Prevents upgrades 24H2:

![ExplorerPatcher](https://github.com/user-attachments/assets/d8577741-dff6-47bd-bd71-b96552d45e65)

Windows 11 Setup
What needs your attention
The following things need your attention to continue the installation and keep your Windows settings, personal files,
and apps.
Why am I seeing this?
ExplorerPatcher
This app can't run because it causes security or performance issues on Windows. A new version may be
available. Check With your software provider for an updated version that runs on this version of Windows.
Learn More

workaround:
1. \Download>` .\ep_setup.exe /uninstall`

2. click refresh Button.

**Comment by sleeper0993** [comment]
Ok, I have a similar issue, but in my case I upgraded to Win 11 24H2 ok (from 23H2) without getting that message. However, explorerpatcher ceases to work. If you look at its properties, it has a message saying it won't work with this version of Windows. Also, one can't uninstall it from the control panel or by using ep_setup.exe /uninstall. When I select either I get a message from the Program Compatability Assistant saying the app can't run because it causes a security or performance issue and it suggests getting a newer version of the app.

**Reply by Amrsatrio** [reply]
Just install the latest version atop of the current one that you have, likely an old version. Also don't forget to add exclusions (if you trust current and future developers of EP). Otherwise download 65.5 again, rename the setup file to ep_uninstall.exe, and run it to stop using ExplorerPatcher.

**Comment by DurgeDriven** [comment]
I don't get this ?

Why not just use a OLD VERSION of explorer patcher.

1. Mine is not recognized as virus  ......if I download new version it is. 
2.  Install old version 
3. Straight away open EP click updates and click don't update
4. You don't need any exclusions whatsoever

That is it nothing else ?
 
I mean if you just want the sleek look of patcher you don't need to update it at all ?  
I understand if you need to use 24H2 right now or you use EP for more things then looks and toolbar then this option is not viable. 
Otherwise 23H2 + old EP is best solution for now imho.

![Screenshot 2024-10-22 064008](https://github.com/user-attachments/assets/1a9bd4f8-4cb1-4092-83dc-c95ae297db72)

**Reply by deeseem** [reply]
Thanks. If the new build isn't compatible, will revert to an older one.

**Reply by pyrates999** [reply]
If you want to use the windows 10 taskbar in windows 11 23h2, with the latest updates applied, you do need the newer versions of EP for that.

**Comment by jav1950** [comment]
i installed window 11 pro version 24H2 but  i can't open Github latest version of
 Valinet explorer patcher it say that it can't run on ths version of windows

**Reply by pyrates999** [reply]
Please post windows version and build.  You can find this by running `winver`.  Please include if you are running x86-64 or Arm.  And what language your windows is in.

If you're running arm, you must install the **ep_setup_arm64.exe** instead of **ep_setup.exe**.
